opus/application/apps/tools/file_utils.py

```diff
@@ -13,7 +13,7 @@
 
 import settings
 
-import pdsviewable
+import pdsfile.pdsviewable
 
```

Comment:
Also need to change line 216 to `viewset = pdsfile.pdsviewable.PdsViewSet.from_dict(preview_json)`